rero_ils/modules/items/api_views.py

```diff
@@ -164,9 +164,29 @@ def automatic_checkin(item, data):
 @jsonify_action
 def cancel_loan(item, params):
     """HTTP request for cancel action."""
+    # TODO: manage transitions for complex cases
     return item.cancel_loan(**params)
 
 
+@api_blueprint.route("/update_loan_pickup_location", methods=['POST'])
+@check_authentication
+def update_loan_pickup_location():
+    """HTTP request for update loan pickup location."""
+    data = flask_request.get_json()
+    loan_pid = data.get('loan_pid')
+    pickup_location_pid = data.get('pickup_location_pid')
+    if not loan_pid:
```

Comment:
1. missing an `if` to check if the `pickup_location_pid` is provided or not. 
2. would be great if you can have the tests coverage green...